### PR TITLE
added `CITATION.cff`

### PR DESCRIPTION
**Pull request by zacharyburnett**
from #59, I populated citation metadata with info from the readme file. It looks like this on the fork:
image

### CODE REVIEW

**Comment by zacharyburnett** [comment]
@cuill @josephzhang8 I couldn't find your Orcids, feel free to add them if you wish

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cff-version: 1.2.0
+message: "If you used this software as part of your work, please use the following citation format."
+authors:
+  - family-names: Calzada
+    given-names: Jaime
```
Comment (zacharyburnett) [inline CITATION.cff:5]:
```suggestion
    given-names: Jaime R
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    orcid: https://orcid.org/0000-0002-3172-1906
+  - family-names: Cui
+    given-names: Linlin
+  - family-names: Zhang
+    given-names: Joseph
+title: "PySchism"
```
Comment (zacharyburnett) [inline CITATION.cff:11]:
```suggestion
title: "PySCHISM"
```

**Comment by jreniel** [comment]
Thanks, Zach!

**Pull request merged**